invenio_cli/commands/requirements.py

```diff
@@ -125,12 +127,13 @@ def check_imagemagick_version(cls, major, minor=-1, patch=-1,
         """Check the ImageMagick version."""
         # Output comes in the form of 'ImageMagick, version 7.0.11-13\n'
         try:
-            result = run_cmd(["magick", "-version"])
-            version = cls._version_from_string(result.output.strip())
-            return cls._check_version(
-                "ImageMagick", version, major, minor, patch, exact
+            get_distribution('wand')
```

Comment:
Hmm.....there's a chicken-egg problem here. This will run in the Invenio-CLI python process where we normally don't install Wand, so it will fail. Now, we run ``check-requirements`` before doing an install, which means that also in the pipenv you wont' have Wand installed. So seems like we need a better solution for detecting if imagemagick is installed.

Reply:
I would give it a try with the ``convert`` command instead

Reply:
So like before just change ``magick`` to ``convert``